kolibri/plugins/coach/assets/src/views/top-nav/index.vue

```diff
@@ -1,32 +1,39 @@
 <template>
 
   <div class="top">
-    <div class="links">
-      <router-link :to="resentPageLink" :class="{active: recentPageActive}" @click.native="blur">
-        {{$tr('recent')}}
-      </router-link>
-      <router-link :to="topicsPageLink" :class="{active: topicsPageActive}" @click.native="blur">
-        {{$tr('topics')}}
-      </router-link>
-      <router-link :to="examsPageLink" :class="{active: examsPageActive}" @click.native="blur">
-        {{$tr('exams')}}
-      </router-link>
-      <router-link :to="learnersPageLink" :class="{active: learnersPageActive}" @click.native="blur" v-if="isDeviceOwner">
-        {{$tr('learners')}}
-      </router-link>
-      <router-link :to="groupsPageLink" :class="{active: groupsPageActive}" @click.native="blur" v-if="isDeviceOwner">
-        {{$tr('groups')}}
-      </router-link>
-    </div>
+    <nav-link
+      :to="recentLink"
+      :active="pageName === Constants.PageNames.RECENT"
```

Comment:
Might be more appropriate to evaluate this in a method? idk. Just seems repetitious.